semanticsearch/test_procedures/semantic-search-common/multiple-subqueries-search.json

```diff
@@ -0,0 +1,196 @@
+      {
+        "operation": "hybrid-query-only-term-range-date",
+        "warmup-iterations": 25,
+        "iterations": 100
+        {%- if not target_throughput %}
+        ,"target-throughput": 6
+        {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
+        {%- else %}
+        ,"target-throughput": {{ target_throughput | tojson }}
+        {%- endif %}
+        {%- if search_clients is defined and search_clients %}
+        ,"clients": {{ search_clients | tojson}}
+        {%- endif %}
+      },
+      {
+        "operation": "bool-only-term-range-date",
+        "warmup-iterations": 25,
+        "iterations": 100
+        {%- if not target_throughput %}
+        ,"target-throughput": 6
+        {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
+        {%- else %}
+        ,"target-throughput": {{ target_throughput | tojson }}
+        {%- endif %}
+        {%- if search_clients is defined and search_clients %}
+        ,"clients": {{ search_clients | tojson}}
+        {%- endif %}
+      },        
+      {
+        "operation": "aggs-query-min-avg-sum",
+        "warmup-iterations": 25,
+        "iterations": 100
+        {%- if not target_throughput %}
+        ,"target-throughput": 6
+        {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
+        {%- else %}
+        ,"target-throughput": {{ target_throughput | tojson }}
+        {%- endif %}
+        {%- if search_clients is defined and search_clients %}
+        ,"clients": {{ search_clients | tojson}}
+        {%- endif %}
+      },
+      {
+        "operation": "aggs-query-min-avg-sum-bool",
+        "warmup-iterations": 25,
+        "iterations": 100
+        {%- if not target_throughput %}
+        ,"target-throughput": 6
+        {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
+        {%- else %}
+        ,"target-throughput": {{ target_throughput | tojson }}
+        {%- endif %}
+        {%- if search_clients is defined and search_clients %}
+        ,"clients": {{ search_clients | tojson}}
+        {%- endif %}
+      },
```

Comment:
We're aiming to simplify this for existing workloads (example [here](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues/133)) and have implemented[ a different approach](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/aa95dce469bc256ec445254929d2523a8ccec5c3/big5/test_procedures/common/big5-schedule.json#L114-L118) as shown in the Big5 workloads. For better user experience, could you simplify this?

Reply:
sure, will check approach you've suggested

Reply:
pushed commit with corresponding change